test/integration/controller_instance_test.go

```diff
@@ -209,6 +209,10 @@ func TestCreateServiceInstanceWithAuthError(t *testing.T) {
 		instance:                     getTestInstance(),
 		skipVerifyingInstanceSuccess: true,
 		preCreateBroker: func(ct *controllerTest) {
+			ct.kubeClient.Lock()
+			t.Log(ct.kubeClient)
```

Comment:
For my own understanding, what is this for and is it related to clusterID or is it just fixing an oversight in the code?

Reply:
debugging. I wanted to print out what the client was, but printing isn't threadsafe.